packages/hash/api/src/graphql/resolvers/pages/updatePageContents.ts

```diff
@@ -162,6 +331,11 @@ export const updatePageContents: Resolver<
       }
     }
 
-    return page.toGQLUnknownEntity();
+    return {
+      page: page.toGQLUnknownEntity(),
+      placeholders: Array.from(placeholderResults.entries()).map(
+        ([placeholderID, entityID]) => ({ placeholderID, entityID }),
```

Comment:
Casing consistency
```suggestion
        ([placeholderId, entityId]) => ({ placeholderId, entityId }),
```

Reply:
Should probably leave as is because of => https://github.com/hashintel/hash/pull/595/files/1a8f78c80337d8389ad8f4a9e8c2dc24091a92d8#r887193923

Reply:
👍